index.js

```diff
@@ -10,7 +10,8 @@ var express = require("express"),
     checkRequest = require('./lib/check-request'),
     filter = require('./lib/filter'),
     q = require('q'),
-    flags = require('flags');
+    flags = require('flags'),
+    {Datastore} = require('@google-cloud/datastore');
```

Comment:
To keep the code more agnostic of the hosting environment, could we have a secrets.json provided by the deploy step, and the code just reads from that?

Reply:
Are you thinking of something like:

1. Add a `create_secrets.js` node file, which pulls from the datastore and sticks the results in `secrets.json`
1. Change `deploy` step in `package.json` to run `create_secrets.js` first before `gcloud app deploy`
1. Add `secrets.json` to `.gitignore`, I guess...

?

My only concern there (and maybe its silly) is that someone might still upload their `secrets.json` file to github someday, by mistake.

Reply:
When `secrets.json` is in `.gitignore` you have to work pretty hard to convince git to track it, so I wouldn't worry too much about that mistake :)